indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskClientSyncImpl.java

```diff
@@ -85,7 +85,11 @@ private boolean stop(final String id, final boolean publish)
           publish ? "publish=true" : null,
           true
       );
-      return isSuccess(response);
+      boolean isSuccess = isSuccess(response);
+      if (!isSuccess) {
+        log.warn("Task [%s] coundln't be stopped because of http request failure.", id);
```

Comment:
is it possible to get the exact http request failure?

Reply:
Added